test/fixtures_stage_based_messaging.js

```diff
@@ -105,16 +105,7 @@ module.exports = function() {
             }
         },
 
-        // 53:
-        {
-            'request': {
-                'method': 'GET',
-                'url': 'http://',
-            },
-            'response': {}
-        },
-
-        // 54: get messagesets
+        // 53: get messagesets
```

Comment:
Please don't adjust the numbers - this can create unnecessary merge problems. Blanks can stay in the fixtures, or they can be used in the future.